### ISSUE TITLE
Hints

### ISSUE THREAD

**Issue by shargon** [facebook/fbctf#421]
How can i do that hints substract the points only when the flag its ok?

Sorry for my english

**Comment by javuto** [comment]
Si quieres puedes repetir la pregunta en español :)

**Comment by shargon** [comment]
Como puedo hacer para que el sistema de hints, penalize solo cuando meten de forma correcta la flag, los puntos bajen, de los posibles que vayan a conseguir, no de los actuales. Lo vemos mas justo 
Gracias :)

**Comment by javuto** [comment]
Cierto, creo que tienes razón. Voy a modificar la issue para hacerlo un poco más claro y para que sea en inglés. Gracias por la sugerencia!

**Comment by javuto** [comment]
Per the discussion above in Spanish with @shargon, the idea is to modify the hint system to take the points from the points that the level is worth, instead of subtracting the points right away, from the points that the team holds. This seems to be more fair and a better improvement to the hints system.
